3rdparty/apriltag/common/matd.cpp

```diff
@@ -786,7 +786,7 @@ static matd_t *matd_op_recurse(const char *expr, int *pos, matd_t *acc, matd_t *
             }
 
             default: {
-                fprintf(stderr, "matd_op(): Unknown character: '%c'\n", expr[*pos]);
+                fprintf(stderr, "matd_op(): Unknownn character: '%c'\n", expr[*pos]);
```

Comment:
@fspindle 

These changes here and below look weird.

Reply:
Thanks, I didn't see that!